### PR TITLE
Fix: hover height value and Add: social links

### PR DESCRIPTION
**Pull request by ChamithSkulathunga**
Check my pull request

### CODE REVIEW

File: R4145/index.html [hunk 1]
```diff
@@ -148,9 +148,9 @@
                 <div>Address : No 123,  ***********.</div>
             </div>
             <div class="social_link">
-                <a href="" class="hr">LinkdIn</a>
-                <a href="" class="h">Twitter</a>
-                <a href="" class="h">GitHub</a>
+                <a href="https://www.linkedin.com/in/chamith-sadeepa-kulathung" target="_blank" class="hr">LinkdIn</a>
```
Comment (radika-insfra) [inline R4145/index.html]: You're used a class name called hr. but it seems that there is no .hr class styles.

File: R4145/index.html [hunk 2]
```diff
@@ -148,9 +148,9 @@
...
+                <a href="https://www.behance.net/chamithsadeepa1" target="_blank" class="h">Behance</a>
```
Comment (radika-insfra) [inline R4145/index.html]: usually we don't use a single letter for class names. since it will make confusions with element selectors. 

.p
p

**Review by radika-insfra — CHANGES_REQUESTED** [review]

**Comment by ChamithSkulathunga** [comment]
ok It was my mistake. I'll fix it. I want to know, after I fix it, what I'm doing again.

**Comment by radika-insfra** [comment]
> ok It was my mistake. I'll fix it. I want to know, after I fix it, what I'm doing again.

after fixing them. you just need to commit and push your changes. it will automatically update the PR.

**Comment by ChamithSkulathunga** [comment]
Sir, I've fixed these errors. Can you check this ?

**Comment by radika-insfra** [comment]
> Sir, I've fixed these errors. Can you check this ?

suer. I'll check. thank you.

**Review by radika-insfra — APPROVED** [review]
Great work

**Pull request merged**